cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -296,17 +265,19 @@ def test_control_crud_ansible_playbook_action(request, catalog_item, action_coll
             "inventory": {
                 "specific_hosts": True,
                 "hosts": ipaddr
-            }
+            },
         }
     view = navigate_to(action, "Edit")
     assert view.description.value == new_descr
     assert view.run_ansible_playbook.inventory.hosts.value == ipaddr
+    view.browser.refresh()
```

Comment:
**Required** We try to avoid using a browser refresh unless absolutely necessary, it increases the length of our test runs significantly. I would suggest renavigating here with the `Force=True` arg.

Reply:
@jawatts Both processes are equal time consuming even re-navigating back to same page will take more time.

Reply:
@sbulage as per my observation... wrong to use `browser refresh` as well `force=True` will totally wrong approach.
I think you are adding work around to land on `Details` page after value assertion right?
If I am right you can use simple approach like
```
view.cancel_button.click()
``` 
Please try and let me know.

Reply:
Trying... let you know.